web/src/pages/Upload.jsx

```diff
@@ -0,0 +1,152 @@
+			method: "POST",
+			body: formData,
+		})
+			.then((res) => {
+				if (!res.ok) throw new Error("Upload failed. Please try again.");
+				return res.json();
+			})
+			.then(() => {
+				setFileList([]); // Clear the file list after successful upload
+				message.success("Upload successful.");
+			})
+			.catch((error) => {
+				message.error(error.message || "Upload failed.");
+			})
+			.finally(() => {
+				setUploading(false);
+			});
+	};
+
+	const props = {
+		onRemove: (file) => {
+			setFileList((prevList) =>
+				prevList.filter((item) => item.uid !== file.uid),
+			);
+		},
+		beforeUpload: (file) => {
+			if (file.type !== "application/zip" && !file.name.endsWith(".zip")) {
+				message.error("You can only upload ZIP files.");
+				return false;
+			}
+			setFileList((prevList) => [...prevList, file]);
+			return true; // Allow the file to be added to the list
+		},
+		fileList,
+		accept: ".zip",
+	};
+
+	return (
+		<div
+			style={{
+				display: "flex",
+				justifyContent: "center",
+				alignItems: "center",
+				height: "100vh",
+				padding: "16px",
+				backgroundColor: "#f4f6f8",
+			}}
+		>
+			<Card
+				style={{
+					width: "100%",
+					maxWidth: 500, // ✅ Responsive fix: works on mobile & desktop
+					textAlign: "center",
+					padding: "24px",
+					borderRadius: "10px",
+					boxShadow: "0px 4px 12px rgba(0, 0, 0, 0.1)",
+				}}
+			>
+				<CloudUploadOutlined style={{ fontSize: "50px", color: "#1890ff" }} />
+
+				{/* 🔽 Updated Title + Instructions 🔽 */}
+				<Title level={3} style={{ marginTop: 16 }}>
+					Upload Your Slack Export ZIP File
+				</Title>
+				<Text type="secondary">
+					This page allows you to upload a ZIP file containing your exported
+					Slack data. If you haven&apos;t exported your Slack workspace data
+					yet, follow the steps below.
+				</Text>
+				<Divider />
+
+				<Title level={5}>How to Export Your Slack Data:</Title>
+				<ul style={{ textAlign: "left", paddingLeft: 24 }}>
+					<li>Go to your Slack workspace settings.</li>
+					<li>
+						Navigate to{" "}
+						<strong>Settings & Administration → Workspace settings</strong>.
+					</li>
+					<li>
+						Find the <strong>Import/Export Data</strong> section.
+					</li>
+					<li>Request an export and download the resulting ZIP file.</li>
+				</ul>
+
+				<Text type="secondary">
+					For detailed instructions, check out{" "}
+					<a
+						href="https://slack.com/help/articles/201658943-Export-your-workspace-data"
+						target="_blank"
+						rel="noopener noreferrer"
+						style={{ marginLeft: 4 }}
+					>
+						this Slack help article.
+					</a>
+				</Text>
+				{/* 🔼 End of Updated Instructions 🔼 */}
+
+				<Divider />
+
+				<Upload {...props} showUploadList={{ showRemoveIcon: true }}>
+					<Button
+						icon={<UploadOutlined />}
+						style={{ width: "100%", marginBottom: 12 }}
+					>
+						Select File
+					</Button>
+				</Upload>
+
+				{/* 🔽 Updated Button (Now Always Enabled) 🔽 */}
```

Comment:
This is interesting. This comment could perhaps be something like:
```suggestion
				{/* 🔽 We deliberately don't disable this button for accessibility reasons 🔽 */}
```

You see how this is more useful?